lib/addons/grafana-operator/index.ts

```diff
@@ -0,0 +1,54 @@
+import { Construct } from 'constructs';
+import merge from "ts-deepmerge";
+import { ClusterInfo, Values } from "../../spi";
+import { createNamespace } from "../../utils";
+import { HelmAddOn, HelmAddOnProps, HelmAddOnUserProps } from "../helm-addon";
+/**
+ * User provided options for the Helm Chart
+ */
+export interface GrafanaOperatorAddonProps extends HelmAddOnUserProps {
+  /**
+   * To Create Namespace using CDK
+   */    
+  createNamespace?: boolean;
+}
+
+/**
+ * Default props to be used when creating the Helm chart
+ */
+const defaultProps: HelmAddOnProps & GrafanaOperatorAddonProps = {
+  name: 'grafana-operator',
+  chart: 'oci://ghcr.io/grafana-operator/helm-charts/grafana-operator',
+  namespace: 'grafana-operator',
+  release: 'grafana-operator',
+  version: 'v5.0.0-rc1',

```

Comment:
Why are we using an RC chart here instead of GA?

Reply:
Thats their latest chart which we were asked to use by GO team. I can change it in future release if they update it. It works very well with this version.